ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -40,6 +40,9 @@ trait ErrorFactories {
   def permissionDenied(description: String): StatusRuntimeException =
```

Comment:
This `trait` has a misleading name, i.e., this is only 1 factory (which creates different types of errors) hence it should be named `ErrorFactory` instead.